src/jquery.stellar.js

```diff
@@ -245,14 +245,16 @@
       this._setScrollTop(oldTop);
     },
     _detectViewport: function () {
-      var viewportOffsets = this.$viewportElement.offset(),
-        hasOffsets = viewportOffsets !== null && viewportOffsets !== undefined;
-
       this.viewportWidth = this.$viewportElement.width();
       this.viewportHeight = this.$viewportElement.height();
-
-      this.viewportOffsetTop = (hasOffsets ? viewportOffsets.top : 0);
-      this.viewportOffsetLeft = (hasOffsets ? viewportOffsets.left : 0);
+      this.viewportOffsetTop = 0;
+      this.viewportOffsetLeft = 0;
+
+      try {
+        var viewportOffsets = this.$viewportElement.offset();
+        this.viewportOffsetTop = viewportOffsets.top;
+        this.viewportOffsetLeft = viewportOffsets.left;
+      } catch(e) { }
```

Comment:
This is the workaround for the `.offset()` changes. Instead of returning `undefined` (as in jQ 1 and 2), calling `$(window).offset()` in jQ 3 can potentially trigger an exception.